indra/util/aws.py

```diff
@@ -99,6 +99,10 @@ def tag_instance(instance_id, **tags):
         logger.info('Adding project tags "%s" to instance %s'
                     % (filtered_tags, instance_id))
         instance.create_tags(Tags=tag_list)
+        vols = instance.volumes.all()
+        for page in vols.pages():
+            for vol in page:
+                vol.create_tages(Tags=tag_list)
```

Comment:
Isn't this supposed to be `create_tags`?

Reply:
Indeed. For some reason I make that typo a lot.